src/components/Dashboard/TodoList.vue

```diff
@@ -83,14 +84,35 @@
                   </div>
                 </div>
 
-                <div v-if="isTypeCorrection(task) && isStatusDraft(task)" class="todo-subtitle">
+                <div v-else-if="isTypeCorrection(task) && isStatusDraft(task)" class="todo-subtitle">
```

Comment:
No. The v-if above this is checks for a subtitle, so THIS line is not an else case of that.

Reply:
Fixed.